app/tests/README.md

```diff
@@ -0,0 +1,19 @@
+# Houston tests
+
+This file contains some technical information for test files.
+
+### Overview
+
+All tests are expected to be ran in parallel and return a promise of an object in the form of:
+```javaScript
+{
+  error: ['error message strings'],
+  warning: ['warning message strings']
+}
+```
+
+NOTE: If you expect to save data to the database, use the `.update()` function instead of `.save()`. This will prevent data overwrites when running tests in parallel.
```

Comment:
True. I didn't want to duplicate the file fetching and parsing from GitHub. I'm not sure what other things we will ever want to save to the database, but it's open in case we want to add things in the future. I'm down for just naming them `components` before we merge.